### PR TITLE
added more details for multi cluster deployment

### PR DESCRIPTION
**Pull request by mikeshng**
Signed-off-by: Mike Ng <ming@redhat.com>

/hold

for feedbacks

### CODE REVIEW

**Review by rokej — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -69,7 +79,7 @@ $ kubectl create ns open-cluster-management-agent-addon
 namespace/open-cluster-management-agent-addon created
 ```
 
-Deploy the subscription add-on on the _hub_ cluster's _managed_ cluster namespace. For example, `cluster1`.
+Deploy the subscription add-on on the _hub_ cluster. For the value of `MANAGED_CLUSTER_NAME`, use the `<cluster name>` that you used for `clusteradm join --cluster-name <cluster name>`.
```
Comment (rokej) [inline README.md:82]: It might be easier for readers if we say ... chose managed cluster names from `kubectl get managedclusters`.

**Review by birsanv — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (mikeshng) [reply README.md]: Done as suggested @rokej please have another look. 

/unhold

Comment (rokej) [reply README.md]: Thank you @mikeshng

**Review by rokej — APPROVED** [review]

**Pull request merged**